packages/web/src/components/play-bar/desktop/PlayBar.jsx

```diff
@@ -95,6 +95,24 @@ class PlayBar extends Component {
       },
       /* throttle= */ 200
     )
+    // Listen for keyboard APIs if available
```

Comment:
Is this the appropriate spot for this logic? I assumed the play bar would make the most sense 🤷